t/run-complex-model-with-filters.t

```diff
@@ -59,24 +59,91 @@ for <hr-inclNOxExtended hr-inclNOxExtendedWithFilters> -> $model-version {
         is (+%output-hash<Storage><tan_into_application>).round(.001), $tan-into-application.round(.001),
                 "Correct tan_into_application result: { (+%output-hash<Storage><tan_into_application>).round(.001) }";
 
-        # check balance for each "Stufe" (livestock, storage, application)
-        # input = output = outflow + loss + remaining  
+        # check balance in N and TAN flows
+        # input = losses + remaining  
         if ($model-version eq "hr-inclNOxExtendedWithFilters") { 
-            ### check livestock ntot balance
-            my $balance-livestock-ntot = %output-hash<Livestock><n_excretion>;
-            my @outputs =
-                'nh3_nlivestock',       # nh3 loss from housing + yard + grazing
-                'n2_ngrazing',          # n2 loss from grazing (housing + yard -> storage)
-                'no_ngrazing',          # no loss from grazing (housing + yard -> storage)
-                'n2o_ngrazing',         # n2o loss from grazing (housing + yard -> storage)
-                'n_remain_grazing',     # ntot remaining in soil from grazing
-                'tan_remain_scrubber',  # ntot remaining (vanishing) in air scrubber
-                'n_out_livestock';      # ntot out of housing + yard + grazing
-            for @outputs -> $output {
-                $balance-livestock-ntot .= apply-pairwise(%output-hash<Livestock>{$output}, &infix:<->, 0);
+
+            ### losses for both, TAN and N
+            
+            ## livestock
+            my $losses-livestock-both = %output-hash<Livestock><n_excretion>.scale(0.0);
+                my @outputs-livestock-both =
```

Comment:
Indentation is wrong. Don't use indentation to visually structure your code, this makes it harder to understand the control flow. Using comments and empty lines to separate "sections" as you already do is fine.